Gemfile.lock

```diff
@@ -91,4 +94,4 @@ DEPENDENCIES
   webrick (~> 1.7)
 
 BUNDLED WITH
-   2.3.10
+   2.4.2
```

Comment:
I am not sure about this version change for the bundler. Here we use 2.3.10: https://github.com/scala/docs.scala-lang/blob/00302d7083b5320c54e65203cf7e16ee94fd3949/Dockerfile#L4

Reply:
Can you give me a brief advice on when to commit Gemfile.lock file change? It seems always is not the option. So is it never? or for some portion of the lock file(e.g. except bundler version?

Reply:
Updated with [fc15ab3](https://github.com/scala/docs.scala-lang/pull/2664/commits/fc15ab3079958eb8354833bf980d39dab2f7be36), would still appreciate your advice.

Reply:
The changes to the lockfile should always be committed, but the file should be generated with the right version of the bundler. Could you please make sure to have bundler 2.3.10 on your system, or to use the Docker approach to automatically use the correct version of the bundler?

Reply:
I actually have bundler 2.3.10 installed at my local mac. Strange it changed the lockfile. 

I will try to commit lockfile with correct bundler version as specified in Dockerfile and will use the Docker approach if I'm not able to do that.